build-coatjava.sh

```diff
@@ -174,6 +178,8 @@ if [ $buildDocs == "yes" ]; then
 fi
 
 # install clara
-#rm -rf clara-home && ./install-clara -c ./coatjava ./clara-home
+if [ $clara == "yes" ]; then
+    rm -rf $clara_home && ./install-clara -c $prefix_dir $clara_home
```

Comment:
DRY violation: `$clara_home` was already removed above, choose one or the other.

Reply:
it's a local variable, want a different name?